common/djangoapps/startup_configurations/validate_config.py

```diff
@@ -0,0 +1,43 @@
+"""
+Common Functions to Validate Configurations
+"""
+from django.conf import settings
+
+
+def validate_lms_config():
+    """
+    Validates configurations for lms and raise ValueError if not valid
+    """
+    validate_common_config()
+
+    # validate feature based configurations
+    validate_marketing_site_config()
+
+
+def validate_cms_config():
+    """
+    Validates configurations for lms and raise ValueError if not valid
+    """
+    validate_common_config()
+
+    # validate feature based configurations
+    validate_marketing_site_config()
+
+
+def validate_common_config():
+    """
+    Validates configurations common for all apps
+    """
+    if not settings.LMS_ROOT_URL:
+        raise ValueError("'LMS_ROOT_URL' is not defined.")
```

Comment:
This function should be safe against bad configurations.  That is, if something's wrong, the only error it can raise should be the one it defines.  Currently, if it is called with a settings file that doesn't have LMS_ROOT_URL defined, it will raise an `AttributeError` instead of the `ValueError`.  I would use `getattr(settings, 'LMS_ROOT_URL', None)` instead.